scalafix-core/shared/src/main/scala/scalafix/internal/rule/Disable.scala

```diff
@@ -131,9 +132,11 @@ final case class Disable(index: SemanticdbIndex, config: DisableConfig)
         val isBlocked = new DisableSymbolMatcher(blockedSymbols)
         ctx.index.symbol(t) match {
           case Some(isBlocked(s: Symbol.Global, disabled)) =>
-            Left(
-              createLintMessage(s, disabled, t.pos)
-            )
+            SymbolOps.normalize(s) match {
```

Comment:
Is it better than the old version? It doesn't affect tests. I added this because of your suggestion to disable normalized symbols.